rest-api/dao/dv_order_dao.py

```diff
@@ -196,6 +191,8 @@ def from_client_json(self, resource_json, id_=None, expected_version=None,
         url=VIBRENT_ORDER_URL).valueString
       if id_ is None:
         order.version = 1
+        order.modified = clock.CLOCK.now()
+        order.created = clock.CLOCK.now()
```

Comment:
After my auto populate fix PR is merged, specifically setting 'created' and 'modified' should not be needed.

Reply:
removed